mapbox/src/main/java/org/odk/collect/mapbox/MapBoxInitializationFragment.kt

```diff
@@ -66,13 +66,17 @@ class MapBoxInitializationFragment : Fragment() {
     }
 
     private fun initializeMapbox(context: Context) {
-        // Auto initialization is disabled to stop x86 devices from crashing so we do this manually
-        context.getState().let {
-            if (!it.get(KEY_MAPBOX_INITIALIZED, false)) {
-                AppInitializer.getInstance(context)
-                    .initializeComponent(MapboxMapsInitializer::class.java)
-                it.set(KEY_MAPBOX_INITIALIZED, true)
+        try {
```

Comment:
I can't work out why this didn't need a `try-catch` when it was called in `MapboxMapFragment`?

Reply:
It wasn't needed because on a device that does not support mapbox choosing mapbox is not possible so that class `MapboxMapFragment` was never used. However `MapBoxInitializationFragment` was always used if there were keys provided (what casued crashes on emulators for example). I improved it in https://github.com/getodk/collect/pull/5141/commits/24af4510bc9f0c43419c47deb8503001ddb2a7ef and now `try-catch` is not needed indeed because we check both keys and supported abis.